RoddyCore/src/de/dkfz/roddy/config/ToolFileGroupParameter.groovy

```diff
@@ -67,8 +68,12 @@ class ToolFileGroupParameter extends ToolEntry.ToolParameter<ToolFileGroupParame
     }
 
     @Override
-    ToolFileGroupParameter clone() {
-        return new ToolFileGroupParameter(scriptParameterName, groupClass, genericFileClass,  passOptions, indexOptions, files?.collect { ToolFileParameter tfp -> tfp.clone() })
+    public ToolFileGroupParameter clone() {
+        List<ToolFileParameter> _files = new LinkedList<ToolFileParameter>()
```

Comment:
It is not a list of files, it is a list of file parameters.
Why did you move away from the collect method?